ledger/participant-state/reference/src/main/scala/com/daml/ledger/participant/state/v1/impl/reference/Ledger.scala

```diff
@@ -356,23 +352,31 @@ class Ledger(timeModel: TimeModel, timeProvider: TimeProvider)(implicit mat: Act
     CommandRejected(Some(submitterInfo), rejectionReason)
 
   private def mkOffset(offset: Int): Offset =
-    Offset(Array(offset))
+    Offset(Array(offset.toLong))
 
-  private def emptyLedgerState = {
-    val heartbeat: Update = Heartbeat(Timestamp.assertFromInstant(timeProvider.getCurrentTime))
-    val stateInit: Update = StateInit(Ref.SimpleString.assertFromString(UUID.randomUUID().toString))
+  override def getLedgerInitialConditions(): Future[LedgerInitialConditions] =
+    Future.successful(
+      LedgerInitialConditions(
+        ledgerId = StateController.getState.ledgerId,
+        recordTimeEpoch = Timestamp.Epoch // FIXME
```

Comment:
Please convert to reference to issue in https://github.com/digital-asset/daml/milestone/13